_posts/2021-08-13-multi-factor-authentication-with-twilio-verify-api.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Multi Factor Authentication: A Developer’s Perspective on Working with Twilio Verify API"
+slug: "multi-factor-authentication-dev-perspective-on-working-with-twilio-verify-api"
+category: blog
+post_image: /assets/images/multi-factor-authentication/header_image.png
+author: seosamh
+date: 2021-08-13 09:30:00
+excerpt: "Multi Factor Authentication is a method of authentication that adds extra steps of verification for gaining access to resources such as an online account, internal system, application or network. Authentication has been traditionally completed by validating a username and password."
+graphic: /assets/images/multi-factor-authentication/header_image.png
+---
+
```

Comment:
Is this feature released or in beta? If it is in beta, then say this and say it is comming soon. Don't want people asking about something they can't access

Also are Support looking for beta testers? If so maybe we can mention this. Maybe check? 

Also a general comment saying something like "Security is of the utmost importance to us here at OnePageCRM.... etc"

Reply:
It is in beta, I will raise this with Sarah to see what she thinks would be the best way of putting this across